config/prte_setup_pmix.m4

```diff
@@ -55,86 +52,28 @@ AC_DEFUN([PRTE_CHECK_PMIX],[
         AC_MSG_ERROR([Cannot continue])
     fi
 
-    AS_IF([test "$with_pmix_extra_libs" = "yes" -o "$with_pmix_extra_libs" = "no"],
+    AS_IF([test "$with_pmix_extra_libs" = "yes" -o "$with_pmixc_extra_libs" = "no"],
```

Comment:
I'm looking, but I don't see "with_pmixc_extra_libs" anywhere. Is this a typo?

Reply:
yes; that's a typo.  sigh.  I'd kill for explicit variable checking :).